### PR TITLE
Add alreday->already

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by lurch** [comment]
Perhaps also `reday->ready` ?

**Comment by luzpaz** [comment]
@lurch added in f2ef772

**Comment by luzpaz** [comment]
@peternewman please review

**Review by peternewman — APPROVED** [review]
LGTM.

Along the same lines you or @lurch could add the other al* typos as just straight ready ones.

**Comment by luzpaz** [comment]
@peternewman done, see 3620559

**Comment by lurch** [comment]
From a quick search in grep.app it seems `reday` is often used in a "Regular Expression for DAY" context, so perhaps that means it ought to be moved to the code dictionary? Sorry for not thinking of checking earlier.

**Comment by luzpaz** [comment]
It's your call.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -21905,8 +21909,11 @@ reasonnable->reasonable
 reasonnably->reasonably
 reassocition->reassociation
 reasssign->reassign
+reasy->ready, easy,
 reatime->realtime
 reattachement->reattachment
+reay->ready, really,
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:21915]: Could also suggest `ray`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -21905,8 +21909,11 @@ reasonnable->reasonable
...
+reayd->ready
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:21916]: Maybe suggest `read` ?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -22219,6 +22227,7 @@ redundacy->redundancy
 redundent->redundant
 reduntant->redundant
 reduse->reduce
+redy->ready
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:22229]: Maybe suggest `red` ?

**Comment by larsoner** [comment]
Merge conflict + unresolved comments

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: dde4437

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: dde4437

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: dde4437

**Comment by luzpaz** [comment]
rebased and added requested revisions in dde4437. If nothing else, she be ready to merge

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -21623,6 +21624,8 @@ radiaton->radiation
 radify->ratify
 radiobuttion->radiobutton
 radis->radix
+rady->ready
+raedy->ready
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:21628]: Sorry for adding more suggestions, but what about
```
raed->read,
raeding->reading,
raeds->reads,
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -22028,8 +22032,11 @@ reasonnable->reasonable
 reasonnably->reasonably
 reassocition->reassociation
 reasssign->reassign
+reasy->ready, easy,
 reatime->realtime
 reattachement->reattachment
+reay->ready, really, red,
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:22038]: `reay->red,` seems like a bit of a stretch? I was suggesting `reay->ray,`

**Comment by luzpaz** [comment]
Added requested revisions in 90957e9

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -22659,6 +22670,7 @@ rekursed->recursed
 rekursion->recursion
 rekursive->recursive
 relaative->relative
+relady->ready
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:22673]: Also `relay` ?

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -21656,6 +21657,11 @@ radiaton->radiation
 radify->ratify
 radiobuttion->radiobutton
 radis->radix
+rady->ready
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:21660]: Perhaps this one should also suggest `ray` too? :man_shrugging:

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -21953,6 +21959,7 @@ readibility->readability
 readible->readable
 readius->radius
 readl-only->read-only
+readly->readily, ready,
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:21962]: Maybe also `deadly`?

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -22061,8 +22068,11 @@ reasonnable->reasonable
 reasonnably->reasonably
 reassocition->reassociation
 reasssign->reassign
+reasy->ready, easy,
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:22071]: I guess you _could_ also suggest `greasy`, but I'm not sure if that's too much of a stretch?

**Pull request merged**